app/controllers/mfa_confirmation_controller.rb

```diff
@@ -49,34 +32,6 @@ def password
     params.require(:user)[:password]
   end
 
-  def handle_valid_password
-    if current_user.auth_app_configurations.any?
-      redirect_to login_two_factor_authenticator_url(reauthn: true)
-    else
-      redirect_to user_two_factor_authentication_url(reauthn: true)
-    end
-    session[:password_attempts] = 0
-    user_session[:current_password_required] = false
-  end
-
-  def handle_invalid_password
-    session[:password_attempts] = session[:password_attempts].to_i + 1
-
-    if session[:password_attempts] < IdentityConfig.store.password_max_attempts
-      flash[:error] = t('errors.confirm_password_incorrect')
-      redirect_to user_password_confirm_url
-    else
-      handle_max_password_attempts_reached
-    end
-  end
-
-  def handle_max_password_attempts_reached
-    analytics.password_max_attempts
-    irs_attempts_api_tracker.logged_in_profile_change_reauthentication_rate_limited
```

Comment:
This looks like the only reference to `IrsAttemptsApi::TrackerEvents#logged_in_profile_change_reauthentication_rate_limited`, should we remove that method?

Reply:
Yes, removed in 486676a